tests/test_data/gtp/gtp.pcf

```diff
@@ -0,0 +1,16 @@
+# clk
```

Comment:
Do we really need PCF files in addition to XDC files now?  I suspect only the XDC files are required now.

Reply:
Not really, but it was to keep consistent with the other tests: https://github.com/SymbiFlow/symbiflow-xc-fasm2bels/blob/5645555c42e94b84fbbcf4e339a223c40aec2010/tests/test_fasm2bels.py#L34

This parametrized test runs both on XDC and PCF